websites/C/Codewars/dist/metadata.json

```diff
@@ -0,0 +1,35 @@
+{
+    "$schema": "https://schemas.premid.app/metadata/1.3",
+    "author": {
+        "name": " .•` KIΛRΛ `•. ",
+        "id": "586905336850677760"
+    },
+    "contributors": [
+        {
+            "name": "ΛΚΘRΛ",
+            "id": "387655649934311427"
+        }
+    ],
```

Comment:
Delete contributors, didn't see anything that he does in commits, also 
![image](https://user-images.githubusercontent.com/9348108/111815807-61cdec00-88dc-11eb-8adc-c31d80227f52.png)